core/chains/evm/txmgr/config.go

```diff
@@ -30,7 +31,10 @@ type Config interface {
 	EvmUseForwarders() bool
 	EvmRPCDefaultBatchSize() uint32
 	KeySpecificMaxGasPriceWei(addr common.Address) *assets.Wei
-	TriggerFallbackDBPollInterval() time.Duration
+
+	// Note: currently only TriggerFallbackDBPollInterval is needed
+	// from here.
+	Database() config.Database
```

Comment:
Note: this broadens the interface quite significantly, since we only need the one method (plus the QConfig methods) from here. I'll follow up with replacing this interface with a struct which just passes in exactly what we need.